pkg/provisioner/dependency.go

```diff
@@ -98,6 +100,14 @@ func kind(tpl *bytes.Buffer, env v1alpha1.Environment) error {
 	return kind.Execute(tpl, env)
 }
 
+func nvkind(tpl *bytes.Buffer, env v1alpha1.Environment) error {
```

Comment:
One thing that's strange here is that all these functions are the same. Why not have a single function for all k8s installers, or pull the differences into these functions?